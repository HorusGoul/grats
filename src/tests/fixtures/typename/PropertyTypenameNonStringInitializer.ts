/** @GQLType */
export class User {
  __typename = 1;
  /** @GQLField */
  name: string = "Alice";
}
