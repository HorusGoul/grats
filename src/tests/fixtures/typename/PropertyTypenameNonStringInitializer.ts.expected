-----------------
INPUT
----------------- 
/** @GQLType */
export class User {
  __typename = 1;
  /** @GQLField */
  name: string = "Alice";
}

-----------------
OUTPUT
-----------------
src/tests/fixtures/typename/PropertyTypenameNonStringInitializer.ts:3:16 - error: Expected `__typename` property initializer to be a string literal. For example: `__typename = "MyType"`.

3   __typename = 1;
                 ~
